tetris.js

```diff
@@ -412,5 +412,34 @@ module.exports = {
 		let newCoordinates = this.rotatesArrayOfCoordinates(tetrominoCoordinates, pivot)
 		let result = this.shiftCoordinates(newCoordinates, shift)
 		return result.sort(this.orderCoordinates);
-	}
+	},
+	incrementPhase: function (typeOfTetromino, currentRotationPhaseOfTetromino) {
+		if(typeOfTetromino === "I" || typeOfTetromino === "S" || typeOfTetromino === "Z"){
+			if(currentRotationPhaseOfTetromino === "vertical"){
```

Comment:
I don't like this implementation.
Use lists and make this method 10 lines or less